### PR TITLE
Presentation link on readme

### LINKED ISSUE #125: Update README with presentation
### Task

- [x] Update readme to include a link to the presentation

### User Story 
As a user, I want to see the presentation from the project repository so I can learn more about it.

### PR DESCRIPTION
**Pull request by FarihaChoudhury**
Added a link to the presentation on the main README, edited authors to add names.

Closes #125

### CODE REVIEW

File: .util/pylint_scores.txt [hunk 1]
```diff
@@ -2,13 +2,11 @@
 ------------------------------------
 Your code has been rated at 10.00/10
 
+************* Module main_page
+dashboard/main_page.py:6:0: E0611: No name 'show_pages' in module 'st_pages' (no-name-in-module)
```
Comment (Zhi-704) [inline .util/pylint_scores.txt:6]: Isn't this a pretty big issue?

File: README.md [hunk 2]
```diff
@@ -99,15 +100,14 @@ There are several directories within the repository to maintain an organised pro
 #### **IMPORTANT**
 >Clicking on the link provided will redirect you to the README in that directory which provides more information
 
-- `archive` - Contains scripts that archive month-old data in the database to avoid over-inflation of data storage. [**Click here**](./archive/README.md).
-- `dashboard` - Contains all the scripts involved in creating and hosting the dashboard. [**Click here**](./dashboard/README.md).
-- `database` - Contains the schema used to create the RDS hosted on AWS as well as any auxiliary functions. [**Click here**](./database/README.md).
+- [`archive`](./archive/README.md). - Contains scripts that archive month-old data in the database to avoid over-inflation of data storage.
```
Comment (Zhi-704) [inline README.md:103]: What happened here?

File: .util/pylint_scores.txt (hunk 1 above)
Comment (FarihaChoudhury) [reply .util/pylint_scores.txt:6]: no Freddy looked at it a couple hrs ago, its a streamlit thing

File: README.md (hunk 2 above)
Comment (FarihaChoudhury) [reply README.md:103]: I just made it so the word is hyperlinked.. do you hate it?

Comment (FarihaChoudhury) [reply README.md:103]: look at the readme preview and lmk I can change it back if you don't like it

Comment (Zhi-704) [reply README.md:103]: Nope it looks good, good shout

**Review by Zhi-704 — APPROVED** [review]

**Pull request merged**